wizeline/remix-project-lab
# Feat like functionality

# What does this PR do?

- Adds LIKE functionality in _project's details page_, adapting it from Project Lab's current Blitz Version
- Adds _.DS_Store_ files to `.gitignore ` file



# Where should the reviewer start?
Check file:
`remix-project-lab/app/routes/projects/$projectId.tsx`
`remix-project-lab/app/models/votes.server.ts`


# How should this be manually tested?

1. Run the Project Locally
2. Go to a projects detail page
3. Use the "LIKE" button
4. First press should add 1 to vote count (Text and icon will change to "UNLIKE")
5. Second click should remove 1 vote from vote count (Text and icon will change to "LIKE")



# Any background context you want to provide?


# What are the relevant tickets?
#37 


# Screenshots
![20221006_remix_plab_feat37_screeshot_2](https://user-images.githubusercontent.com/90728303/194446174-ba31337b-3bb6-4762-9d90-84c7449e3456.png)
![20221006_remix_plab_feat37_screeshot_1](https://user-images.githubusercontent.com/90728303/194446177-85df4a57-e4b8-41c1-b2b5-3f7f261b9258.png)



# Questions



# Checklist
